### ISSUE TITLE
How do we make this so people and use the things we build easily?

### ISSUE THREAD

**Issue by benjaminkay93** [NaSTAuk/graphics#2]
Monorepos / tech projects arn't easy for people to use, but we want people to be capable of setting up a config with the graphics they would like easily and with minimal effort and coding.

Making this happen aint gonna be easy, ideas would be nice :D

**Comment by moschopsuk** [comment]
My personal opinion that one using this will in most scenarios be the same one setting up CasparCG or OBS.

So, it would be fair to assume they will have some technical understanding about this. But nevertheless, this should be well documented with various install and setup guides.

**Comment by benjaminkay93** [comment]
yeah i can imagine it depends alot on the outcome of #4 too. I have some ideas about how to use but i dont want to solutionise too soon aha
